src/monitor/image_loader.c

```diff
@@ -194,11 +196,9 @@ long load_zstd_img(const char *filename){
 
 #endif  //  CONFIG_MEM_COMPRESS
 
-// Return whether a file is a gz file, determined by its name.
-// If the filename ends with ".gz", we treat it as a gz file.
-
-
-long load_img(char* img_name, char *which_img, uint64_t load_start, size_t img_size) {
+// Will check the magic number in file, if not gz or zstd archive,
+// will read in as a raw image
+long load_img(char* img_name, char *which_img, uint8_t* load_start, size_t img_size) {
```

Comment:
i fix this comment, i think you might be talking about this? Or am I misunderstanding?

Reply:
ohhhh, ok